spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactoryTests.java

```diff
@@ -381,7 +384,8 @@ void testCreateSubscriber_validateSetProperties() {
         .hasFieldOrPropertyWithValue("maxAckExtensionPeriod", Duration.ofSeconds(2L))
         .hasFieldOrPropertyWithValue("minDurationPerAckExtension", Duration.ofSeconds(3L))
         .hasFieldOrPropertyWithValue("maxDurationPerAckExtension", Duration.ofSeconds(4L))
-        .hasFieldOrPropertyWithValue("numPullers", 2);
+        .hasFieldOrPropertyWithValue("numPullers", 2)
+        .hasFieldOrPropertyWithValue("subStubSettings.endpoint", "test.endpoint");
```

Comment:
It may also be worth covering this in our integration tests? https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/f937b36efb442e5294a357c1a5a14e529aac3fa4/spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/pubsub/it/PubSubAutoConfigurationIntegrationTests.java#L46

Reply:
Added and also found some unrelated bug accidentally.